test/loopback.test.js

```diff
@@ -99,9 +100,72 @@ describe('loopback', function() {
         'version',
       ];
 
+      var EXPECTED_GTNODE4 = [
```

Comment:
correct .. I didn't realize that we shouldn't worry anymore about pre4.0 so I fixed that. 
`arguments` and `caller` properties are removed though! Possibly because of the use strict.